MartinP460/react-denmark-map
# [V2] Add ESLint import order rules and use absolute paths

### Proposed changes

Added ESLint rules for sorting imports. Also made the core package use absolute imports with the prefix `@`.

### How to test

1. In any file with multiple imports, try to switch up the import order and verify that ESLint complains.
2. Similarly, try using a relative import and make sure ESLint complains.
